src/components/theme/Widgets/Login.stories.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Login from '../Login/Login';
+import Wrapper from '@plone/volto/storybook';
+
+const { Provider } = require('react-intl-redux');
+const configureStore = require('redux-mock-store').default;
+const store = configureStore()({
+  userSession: {
+    login: {},
+  },
+  intl: {
+    locale: 'en',
+    messages: {},
+  },
+});
+
+const StoryComponent = (args) => {
+  return (
+    <Provider store={store}>
+      <Wrapper>
+        <Login />
+      </Wrapper>
+    </Provider>
+  );
+};
+
+export const Default = StoryComponent.bind({});
+Default.args = {
+  login: () => null,
+};
+
+export default {
+  title: 'Internal Components/Login',
```

Comment:
@danalvrz Let's put both this and the PreviewImage story in the "Public Components" section instead of "Internal Components", since they are things that can be seen by users who have not logged in yet, and not only by editors.